model/riscv_sys_control.sail

```diff
@@ -106,15 +254,136 @@ function check_Counteren(csr : csreg, p : Privilege) -> bool =
     (0xC00, Supervisor) => mcounteren.CY() == 0b1,
     (0xC01, Supervisor) => mcounteren.TM() == 0b1,
     (0xC02, Supervisor) => mcounteren.IR() == 0b1,
+    (0xC03, Supervisor) => mcounteren.HPM()[0] == bitone,
```

Comment:
Is there any difference between `0b1` and `bitone`? If not it's probably best not to mix the two styles here.